### DISCUSSION TITLE
NI 2025 Beta Release 2

### DISCUSSION THREAD

**Discussion by jhershbe** [wpilibsuite/2025Beta#21] [Announcements]
This release includes roboRIO Image version 1.1

Fixes linkage issues with NetworkTables from LabVIEW

<hr /><em>This discussion was created from the release <a href='https://github.com/wpilibsuite/2025Beta/releases/tag/NI_GAME_TOOLS_BETA_2'>NI 2025 Beta Release 2</a>.</em>

**Comment by ArchdukeTim** [comment]
Receiving this error: 

> A package you selected, or one of its dependencies, requires the latest version of Package Manager to continue with installation.

**Reply by ArchdukeTim** [reply]
https://github.com/wpilibsuite/2025Beta/issues/9
